bleak/backends/p4android/scanner.py

```diff
@@ -61,6 +61,41 @@ def __init__(
     def __del__(self) -> None:
         self.__stop()
 
+    async def request_permissions(self, **kwargs) -> None:
+        loop = asyncio.get_running_loop()
+        permission_acknowledged = loop.create_future()
+        def handle_permissions(permissions, grantResults):
+            if any(grantResults):
+                loop.call_soon_threadsafe(
+                    permission_acknowledged.set_result, grantResults
+                )
+            else:
+                loop.call_soon_threadsafe(
+                    permission_acknowledged.set_exception(
+                        BleakError("User denied access to " + str(permissions))
+                    )
+                )
+        if not kwargs.get("permissions"):
```

Comment:
Instead of kwargs, I would just make a named arg that defaults the default permissions.

Reply:
done